EAGLE_SIP/BeagleBone_Blue.sch

```diff
@@ -17715,23 +17715,13 @@ Attribution-ShareAlike 3.0 United States License.
 <wire x1="172.72" y1="35.56" x2="160.02" y2="35.56" width="0.1524" layer="91"/>
 <label x="161.29" y="35.56" size="1.778" layer="95"/>
 </segment>
-<segment>
-<pinref part="CAN" gate="G$1" pin="4"/>
```

Comment:
I had to remove a lot of stuff to get the schematics / board to sync since I didn't notice it sooner.  The CAN is added back in a later commit.